backend/inspirehep/records/marshmallow/literature/bibtex.py

```diff
@@ -66,17 +66,24 @@ class BibTexCommonSchema(Schema):
 
     @staticmethod
     def get_date(data, doc_type):
+        from inspirehep.records.api import LiteratureRecord
```

Comment:
you should put imports at the module scope in principle

Reply:
in that case there is an error, caused by (I guess) circular import, so when I move it on the top it does not work

Reply:
I see. Actually you don't need this, as `data` should already be a `LiteratureRecord` in real cases. You should use the `create_record` fitxure in the tests to build a record.